bundles/org.eclipse.equinox.app/META-INF/MANIFEST.MF

```diff
@@ -2,29 +2,30 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.equinox.app; singleton:=true
-Bundle-Version: 1.6.100.qualifier
+Bundle-Version: 1.6.200.qualifier
 Bundle-Vendor: %providerName
 Bundle-Activator: org.eclipse.equinox.internal.app.Activator
 Bundle-Localization: plugin
 Export-Package: 
  org.eclipse.equinox.app;version="1.1",
- org.eclipse.equinox.internal.app;x-friends:="org.eclipse.core.runtime",
- org.osgi.service.application;version="1.1"
+ org.eclipse.equinox.internal.app;x-friends:="org.eclipse.core.runtime"
 Import-Package: org.eclipse.osgi.framework.console;resolution:=optional,
  org.eclipse.osgi.framework.log,
  org.eclipse.osgi.service.datalocation,
  org.eclipse.osgi.service.debug,
- org.eclipse.osgi.service.environment; version="1.1",
+ org.eclipse.osgi.service.environment;version="1.1",
  org.eclipse.osgi.service.runnable,
  org.eclipse.osgi.storagemanager,
  org.eclipse.osgi.util,
  org.osgi.framework;version="1.3",
- org.osgi.service.condpermadmin; resolution:=optional,
+ org.osgi.service.application;version="[1.1.0,1.2.0)",
+ org.osgi.service.condpermadmin;resolution:=optional,
  org.osgi.service.event;version="1.0.0";resolution:=optional,
  org.osgi.service.packageadmin;version="1.2",
  org.osgi.util.tracker
 Require-Bundle: org.eclipse.equinox.registry;bundle-version="[3.4.0,4.0.0)",
- org.eclipse.equinox.common;bundle-version="[3.2.0,4.0.0)"
+ org.eclipse.equinox.common;bundle-version="[3.2.0,4.0.0)",
+ org.osgi.service.application;bundle-version="[1.1.0,1.2.0)";visibility:=reexport
```

Comment:
Don't import package AND require bundle. One or the other.

Reply:
Here it has to be require bundle because we need the `reexport` to provide the package from the required bundle.  Drop the `Import-Package` of `org.osgi.service.application`

Reply:
Yes. I think I wanted to prepare for the time when the Require-Bundle is removed. But who knows if/when this happens and then it can be switched and there is indeed no point in having both now.
Removed it.